message DescribeRequest {
    required bool verbose = 1;
}